Source/MMWormholeSession.m

```diff
@@ -0,0 +1,137 @@
+//
+// MMWormholeSession.m
```

Comment:
I added a separate branch with all the project changes called watchOS_example. I wanted to keep this PR a little cleaner for now to focus on the core changes.